package/ports_daemon/ports_daemon/src/can.c

```diff
@@ -27,6 +28,7 @@
 /* can_settings_initialized prevents false warning when notify function
  * is triggered by read from persistent config file during boot */
 static bool can_settings_initialized = false;
+static bool is_duro = device_is_duro();
```

Comment:
I would prefer we initialize to sane default like `false` and initialize in `can_init`